.github/workflows/reusable-release.yml

```diff
@@ -0,0 +1,186 @@
+name: Pull Request - Build and Deploy
+on:
+  workflow_call:
+    inputs:
+      componentName:
+        description: The component name using the workflow
+        required: true
+        type: string
+      dockerTag:
+        description: The Tag to be associated with the new docker image (optional)
+        required: false
+        type: string
+      releaseNotes:
+        description: The Release Notes associated with the Release (optional)
+        required: false
+        type: string
+      releaseVersion:
+        description: The version to be Released (#.#.#.#)
```

Comment:
Change to #.#.#, we only want the 4th portion for hotfixes when we can't do a patch, don't want to encourage the use of 4 portions otherwise.

Reply:
Done and added in format for RC